tests/test_positional_channels.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+import torch
+import xarray as xr
+
+from ocean_emulators.config import SamudraConfig
+from ocean_emulators.constants import TensorMap
+from ocean_emulators.models.samudra import Samudra
+from ocean_emulators.utils.data import DataSource, Normalize
+from ocean_emulators.utils.multiton import MultitonScope
+
+
+def test_positional_parameters_update():
+    h, w = 4, 5
+    with MultitonScope():
+        TensorMap.init_instance("thetao_1", "hfds")
+        coords = {"lev": [0], "y": np.arange(h), "x": np.arange(w)}
+        zero = np.zeros((1, h, w))
+        data = xr.Dataset(
+            {
+                "thetao": (("lev", "y", "x"), zero),
+                "hfds": (("y", "x"), np.zeros((h, w))),
+            },
+            coords=coords,
+        )
```

Comment:
Make it a little more consistent with the `ones` example below.

```suggestion
        data = xr.Dataset(
            {
                "thetao": (("lev", "y", "x"), np.zeros((1, h, w))),
                "hfds": (("y", "x"), np.zeros((h, w))),
            },
            coords=coords,
        )
```